packages/askui-nodejs/src/execution/ui-control-client.ts

```diff
@@ -574,67 +555,57 @@ export class UiControlClient extends ApiCommands {
    * **Examples:**
    * ```typescript
    * // Click text that matches exactly
-   * await aui.clickText({text: 'askui', exact: true})
+   * await aui.clickText({text: 'askui', type: 'similar'})
    *
    * // Click text that contains 'pie' or 'cake' or 'Pie' or 'Cake'
-   * await aui.clickText({text: '.*([Pp]ie|[Cc]ake).*', regex: true})
+   * await aui.clickText({text: '.*([Pp]ie|[Cc]ake).*', type: 'regex'})
    *
    * // Click the text 'TERMINAL' that is left of the text 'Ports'
-   * await aui.clickText({text: 'TERMINAL', relation: {type: 'leftOf', text: 'PORTS'}})
+   * await aui.clickText({
+   *     text: 'TERMINAL',
+   *     type: "exact",
+   *     relation: { type: 'leftOf', text: 'PORTS' }
+   *   })
    * ```
-   *
    * @param {Object} params - Object containing required `text` property and optional properties
    *                          for regular expression matching and relation.
    * @property {string} params.text - The text to be clicked.
-   * @property {boolean} [params.regex=false] - Whether the text is matched using a regular
-   *                                            expression, default is false.
-   * @property {boolean} [params.exact=false] - Whether an exact match of the text is required
-   *                                            when using regex, default is false.
-   * @property {Object} [params.relation] - Object describing the relationship between the
-   *                                        clicked text and another element.
-   * @property {Relations} params.relation.type - The type of relation.
+   * @property {string} params.type - Whether the text is matched using similarity,
+   *                                  exact match or a regular expression.
+   * @property {Object} params.relation - Object describing the relationship between the
+   *                                      clicked text and another element.
+   * @property {RelationsForConvenienceMethods} params.relation.type - The type of relation.
    * @property {string} params.relation.text - The label or text associated with the
    *                                           related element or state.
    */
   async clickText(
     params: {
       text: string,
-      regex?: boolean,
-      exact?: boolean,
-      relation?: { type: Relations, text: string }
+      type: 'similar' | 'exact' | 'regex',
```

Comment:
What about making it optional or moving it below `relation` so you can add `'similar'` as the default?